src/requestLib.js

```diff
@@ -108,7 +108,8 @@ var RequestLib = {
         });
   
         clientRequest.on('end', function () {
-          RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_END, params.request.method + ' ' + requestOptions.path, false, {
+          var url = requestType === constants.REQUEST_TYPES.PROXY ? clientRequest.url : "http://" + clientRequest.headers.host + clientRequest.url;
```

Comment:
is the concatenation correct? host + url?

Reply:
in case of proxy request URL has only path

Reply:
used requestOptions.path

Reply:
can be resolved...